homeassistant/components/dlib_face_identify/image_processing.py

```diff
@@ -14,44 +17,121 @@
     PLATFORM_SCHEMA,
     ImageProcessingFaceEntity,
 )
-from homeassistant.core import split_entity_id
+from homeassistant.core import callback, split_entity_id
 import homeassistant.helpers.config_validation as cv
 
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_NAME = "name"
+ATTR_DISTANCE = "distance"
 CONF_FACES = "faces"
 
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Required(CONF_FACES): {cv.string: cv.isfile},
+        vol.Required(CONF_FACES): vol.Any({cv.string: cv.isfile}, cv.isdir),
```

Comment:
I feel it would be more consistent to have {cv.string: cv.isfile}, {cv.string: cv.isdir}

Reply:
I intentionally made it this way as I felt that either having a mix of files/directories or multiple directories in the configuration could be confusing. "{cv.string: cv.isfile}" is there purely for backward compatibility.

Reply:
This small change is ok in regards to ADR7.